config/settings/base.py

```diff
@@ -279,3 +279,6 @@ def immutable_file_test(path, url):
 
 
 WHITENOISE_IMMUTABLE_FILE_TEST = immutable_file_test
+
+S3_FILE_BUCKET = env.str('DJANGO_S3_FILE_BUCKET')
```

Comment:
I'd recommend replacing these with a single URI.
This would allow the use of "/tmp/data" for local, and "s3://my_bucket/some/path" down the line.

Reply:
Haven't implemented local storage yet (as that would require an endpoint to accept file data, instead of just metadata) but have generalized to remove references to bucket, and detect s3:// urls.